app/views/allies/index.html.erb

```diff
@@ -31,7 +31,19 @@
     </div>
   </div>
 <% end %>
-
+<% if @invited_allies.any? %>
+  <div class="title"><%= t('allies.index.invited') %></div>
+  <div class="spacer"></div>
+  <div class="gridThree">
+  <% @invited_allies.each do |ally| %>
+    <div class="gridThreeItemBoxLight ally">
+      <div><%= mail_to "#{ally.email}" %></div>
+      <div class="subtle"><%=t('allies.index.invited_on', date: format_date(ally.invitation_sent_at)) %></div>
+      <div class="subtle"><%= t('allies.index.invitation_expired') %> <%= ally.invitation_sent_at < 2.weeks.ago ? t('yes_text') : t('no_text') %></div>
```

Comment:
We should only show this if invitation_accepted_at doesn't exist as well. I think if an invitation is accepted within 2 weeks, it will technically show up here.

I think it's useful just to show pending invitations rather than accepted ones.

Reply:
Thanks for taking a look at this so quickly! 
I included `invitation_accepted_at: nil` in the query:
https://github.com/ifmeorg/ifme/blob/393edba96b2a9f9da120ec3a44cc20e1e4ea268b/app/controllers/allies_controller.rb#L14-L16
so I don't think they'll show up